### PR TITLE
[369] allow support users to edit confirmed allocation numbers

### PR DESCRIPTION
**Pull request by stellenberger**
### Context

- [This trello ticket](https://trello.com/c/GzEv6eFo/369-allow-support-users-to-edit-confirmed-allocation-numbers)

### Changes proposed in this pull request

- Add edit/update functionality to change allocation numbers on the Support console

### Guidance to review

- Go to the support console
- Click on the PE Allocations tab
- Choose a provider
- Click the Change link on the allocation number

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: spec/support/page_objects/support/allocations_index.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Support
+    class AllocationsIndex < PageObjects::Base
+      set_url "/support/allocations"
+
+      def providers
+        page.find_all(".qa-provider_row")
+      end
```
Comment (defong) [inline spec/support/page_objects/support/allocations_index.rb:10]: why not `elements` or `sections`?

**Comment by stellenberger** [comment]
[Allow support users to edit confirmed allocation numbers](https://trello.com/c/GzEv6eFo/369-allow-support-users-to-edit-confirmed-allocation-numbers)

**Review by malcolmbaig — CHANGES_REQUESTED** [review]
Leaving a note for @stellenberger — please double check with @mbocc if we still need this functionality.

**Comment by stellenberger** [comment]
Closing this as feature no longer needed

**Pull request closed without merging**